### PR TITLE
Summarizing the total click of website links

### PR DESCRIPTION
**Pull request by vinesh-sn**
[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | No
| New feature? | Yes
| Related user documentation PR URL |  https://github.com/mautic/documentation/pull/95
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | NA
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Here we are summarizing click statistics in each email

#### Steps to test this PR:
1.  Select Email from the Channel menu
2.  Select any one of the email's detail view
3. At the bottom of the page, we will have a click count table for each links in that email
4. The new feature is, now we can see a total clicks column which displays the sum of all the clicks 
 
[//]: # ( As applicable: )

### CODE REVIEW

**Comment by npracht** [comment]
This is great. A piece of https://github.com/mautic/mautic/issues/1883 is done.
Now for the great marketers we just miss to be able to click on the number to know who clicked in the email :-)

File: app/bundles/PageBundle/Views/Trackable/click_counts.html.php [hunk 1]
```diff
@@ -10,14 +10,29 @@
...
-            <?php endforeach; ?>
+                
+                <?php foreach ($trackables as $link): ?>
+                    <tr>
+                        <td class="long-text"><a href="<?php echo $link['url']; ?>"><?php echo $link['url']; ?></a></td>
+                        <td class="text-center"><?php echo $link['hits']; ?></td>
+                        <td class="text-center"><?php echo $link['unique_hits']; ?></td>
+                        <td><?php echo $link['redirect_id']; ?></td>
+                    </tr>
+                <?php endforeach; ?>
```
Comment (dongilbert) [inline app/bundles/PageBundle/Views/Trackable/click_counts.html.php]: You can avoid iterating the `$trackables` twice by putting the total clicks row at the end of the table, and move the counting logic from lines 14-20 into the foreach beginning on line 28.

Also, "Total Clicks" should be translated. You can replace it with `$view['translator']->trans('mautic.core.total_clicks');`, then add that translation string into the `app/Bundles/CoreBundle/Translations/en_US/messages.ini` with a value of "Total Clicks"

Comment (vinesh-sn) [inline app/bundles/PageBundle/Views/Trackable/click_counts.html.php]: @dongilbert The correction is done. Thank you :-)

**Comment by escopecz** [comment]
:+1:

**Pull request merged**